internal/provider/resource_auth0_attack_protection.go

```diff
@@ -396,6 +396,7 @@ func flattenBreachedPasswordProtection(bpd *management.BreachedPasswordDetection
 			"enabled":                      bpd.GetEnabled(),
 			"method":                       bpd.GetMethod(),
 			"admin_notification_frequency": bpd.GetAdminNotificationFrequency(),
+			"shields":                      bpd.GetShields(),
```

Comment:
@nialdaly This is what was causing the tests to fail 🙈 we erroneously removed the shields property when we added the pre_user_registration block

Reply:
ah my bad! Good catch haha